static/js/event/event.ts

```diff
@@ -18,14 +18,49 @@
  * Disaster events
  */
 
+import axios from "axios";
 import React from "react";
 import ReactDOM from "react-dom";
 
-import { Page } from "./page";
+import { EventPage, Page } from "./page";
+import { Property } from "./types";
+
+function parseApiResponse(resp): Array<Property> {
+  // Parses response from triples API for later rendering
```

Comment:
when adding comments for what a function does, we want to add function comments above the function like: 

```
/**
 * Parses response from ...
 */
function parseApiResponse ...

Reply:
Done!